conf/mysql_hook_regen_conf

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+force=${2:-0}  # 0/1 --force argument
+dryrun=${3:-0}  # 0/1 --dry-run argument
+pending_conf=$4 # Path of the pending conf file
+
+
+do_pre_regen() {
+  sed --in-place "s@thread_stack = 128K@thread_stack = 192K@g" "${pending_conf}/../mysql/etc/mysql/my.cnf"
```

Comment:
(Note that it should become irrelevant once we're on bullseye because it will rely on the default mysql conf instead of our super-old custom one)

Reply:
In that case, is it better to remove all that `my.cnf` tweaking ?